lib/Doctrine/ORM/Tools/SchemaTool.php

```diff
@@ -871,4 +857,21 @@ public function getUpdateSchemaSql(array $classes, $saveMode = false)
 
         return $schemaDiff->toSql($this->platform);
     }
+
+    private function gatherColumnOptions(array &$options, array $mapping)
+    {
+        if (isset($mapping['options'])) {
+            $knownOptions = array('comment', 'unsigned', 'fixed', 'default');
```

Comment:
I don't think it makes sense to copy the column comment from the PK to the foreign key. So that should probably be excluded, but makes it kinda more ugly.